### DISCUSSION TITLE
Number of snakes in the Fall gold and platinum tiers seems wrong

### DISCUSSION THREAD

**Discussion by af7567** [BattlesnakeOfficial/feedback#132] [Bug Reports]
I was just looking at the Fall League leaderboard after the changes to the points thresholds and noticed that platinum has 17 snakes, and gold has 33 snakes. Is this just because the elite tier isn't full yet, or are the calculations off by 1?

**Comment by bvanvugt** [comment]
Indeed! Thanks for reporting this, we'll get it fixed shortly.

**Comment by bvanvugt** [comment]
Fixed in version 1.6.46!
